File: 242-valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!=len(t):
            return False
        s_sort=''.join(sorted(s))
        t_sort=''.join(sorted(t))
        if(s_sort==t_sort):
            return True
        return False


        

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int freq[]=new int[26];
        if(s.length()!=t.length()){
            return false;
        }
        for(int i=0;i<s.length();i++){
            freq[s.charAt(i)-'a']++;
        }
        for(int j=0;j<s.length();j++){
            freq[t.charAt(j)-'a']--;
        }
        for(int cnt:freq){
            if(cnt!=0){
                return false;
            }
        }
        return true;
    }
}